### ISSUE TITLE
How to show full property description instead abridged one on class page?

### ISSUE THREAD

**Issue by Rubydesic** [Kotlin/dokka#3975]
**Question**
Dokka only shows an abridged description of the property on the documentation page for the class that contains it. How can I configure it to show the full description?

**Screenshots**
The class page shows an abridged description of the property:
![image](https://github.com/user-attachments/assets/de2a011b-2900-48f1-b369-e476cb0bc4ae)

The full description of the property:
![image](https://github.com/user-attachments/assets/c852e716-24a4-4405-ac4a-a30a7def05f1)

**Installation**
- Operating system: Windows
- Build tool: Gradle
- Dokka version: 2.0.0

**Comment by vmishenev** [comment]
According to [the description of KDoc](https://kotlinlang.org/docs/kotlin-doc.html), Dokka displays only the first paragraph of the documentation text (the block of text until the first blank line) for members on a class page.
There is no way to display the full documentation for members.

Could you elaborate on your use case? Do you want to show the full documentation for all members (properties/functions)? Do you not need their dedicated pages?"

**Comment by Rubydesic** [comment]
Yes, I want to show the full documentation for all members and not have dedicated pages. In a [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html), for example, one can read the full documentation for each member without needing to click on links. 

In my opinion, requiring the reader to click on links to see each member's information makes the documentation annoying to read and reduces discoverability.
